### DISCUSSION TITLE
Invalid string length error in snapshot tests

### DISCUSSION THREAD

**Discussion by alperunal** [vuejs/test-utils#1372] [Q&A]
Hello,

When I add **$store** to mocks, I get the following error: 
Snapshot test
RangeError: Invalid string length

Any idea?

```
  it('Snapshot test', () => {
    wrapper = shallowMount(MyComponent, {
      global: {
        mocks: {
          $store: {
            state: {
              details: {
                someArray: [],
                someObject: {
                  name: '',
                  surname: '',
                },
              },
            },
            commit: jest.fn(),
          },
        },
      },
      props: {
        title: 'MyTitle',
      },
    });
    expect(wrapper).toMatchSnapshot();
  });
```

edit: Same issue with adding store in plugins array.

**Comment by cexbrayat** [comment]
Hi @alperunal 

That's weird. WOuld you mind opening an issue and providing a small repro so we can investigate? You can use `create-vue` to have a minimal project with Vitest and VTU. Make sure you use the latest versions, and if the issue persists then we'll take a look.
